programs/lib/syscalls/syscalls.c

```diff
@@ -128,18 +121,26 @@ int _rename (const char *oldpath, const char *newpath) {
     return -1;
 }
 
-// FIXME: This should not be actually called because of tinyalloc.
+// Based on the implementation found here: https://sourceware.org/git/gitweb.cgi?p=newlib-cygwin.git;a=blob;f=libgloss/aarch64/syscalls.c
+// with some modifications
 void *_sbrk(int incr) {
```

Comment:
Had to modify this slightly to work with newlib's malloc, since malloc uses this function under the hood.